plugins/crypto/certstores/ua_certificategroup_memorystore.h

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2024 (c) Fraunhofer IOSB (Author: Noel Graf)
+ */
+
+#ifndef UA_CERTIFICATEGROUP_MEMORYSTORE_H
+#define UA_CERTIFICATEGROUP_MEMORYSTORE_H
+
+#include <open62541/plugin/certificategroup.h>
+
+struct MemoryCertStore;
+typedef struct MemoryCertStore MemoryCertStore;
+
+struct MemoryCertStore {
```

Comment:
I think most of the structure can be replaced with a UA_TrustListDataType.
Implement UA_TrustListDataType_add, UA_TrustListDataType_remove.
Both taking two trust lists as the input.
This can go into /open62541/util.h as it is generally useful.

In the end we will have a trust list and a rejected list. That is all.
This is so little code that we should fold it into the security group and not keep it separate.